introduction.md

```diff
@@ -1,2 +1,8 @@
 an introduction
-Alright, let's lie =/ "Electives"
\ No newline at end of file
+Alright, let's lie =/ "Electives"
+
+Deadly skills
+=============
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?